packages/server-core/src/user/identity-provider/identity-provider.class.ts

```diff
@@ -155,16 +158,30 @@ export class IdentityProvider extends Service {
     )
     // DRC
     try {
-      if (result.user.userRole !== 'guest') {
+      if (result.user.userRole === 'user' || result.user.userRole === 'admin') {
```

Comment:
lets keep this as `result.user.userRole !== 'guest'` to ensure forward compatability with other potential roles

Reply:
Changed in current version